common/build/eslint-config-fluid/src/test/mockFiles/no-member-release-tags/mockClassDeclaration.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @public
+ */
+class MockClass {
+	/**
+	 * @internal
+	 */
+	invalidInternal(): void {}
+
+	/**
+	 * @alpha
+	 */
+	invalidAlpha(): void {}
+
+	/**
+	 * @beta
+	 */
+	inValidBeta(): void {}
+
+	/**
+	 * @public
+	 */
+	inValidPublic(): void {}
+
+	//@public
+	invalidLineComment(): void {}
+
+	/**
+	 * Correctly implemented method with valid comment.
+	 */
+	validBlockComment(): void {}
+
+	// Correctly implemented method with a line comment.
+	validLineComment(): void {}
+
+	validNoComment(): void {}
+
+	signature: string;
+}
+
+/**
+ * @public
+ */
+class MockClassTwo {
```

Comment:
Like with the interface, not sure we gain something out of having two classes. All the cases we want to validate could be inside a single class object, right?

Reply:
This also has a somewhat _similar_ reasoning as the inclusion of release tag for the parent node. 

More often than not, multiple classes live in a single file, thus I wanted to verify that the linter is able to pick multiple parent nodes inside a single file.

Reply:
Let's add a comment to the TSDoc bit for the second class that says that, so down the road people can know what the intention is.